rules/windows/command_and_control_rmm_netsupport_susp_path.toml

```diff
@@ -0,0 +1,102 @@
+
+- Review the origin of the NetSupport file and if it's related to an authorized IT Support case.
+- Examine if the NetSupport process is set to persist in the affected system via scheduled task, Startup folder or Run key.
+- Check the network, files and child processes activity associated with the NetSupport client32.exe process.
+- Correlate the event with other security alerts or logs from data sources like Elastic Defend or Microsoft Defender for Endpoint to gather additional context and identify any related malicious activities.
+
+### False positive analysis
+
+- Legitimate IT support activities using NetSupport by IT support accounts.
+
+### Response and remediation
+
+- Immediately isolate the affected system from the network to prevent further unauthorized access or lateral movement by the attacker.
+- Terminate any suspicious processes identified in the alert, such as PowerShell, cmd.exe, or other flagged executables, to halt any ongoing malicious activity.
+- Review and revoke any unauthorized user accounts or privileges that may have been created or modified using tools like net.exe or schtasks.exe.
+- Conduct a thorough scan of the affected system using endpoint protection tools to identify and remove any malware or unauthorized software installed by the attacker.
+- Restore the system from a known good backup if any critical system files or configurations have been altered or compromised.
+- Escalate the incident to the security operations center (SOC) or incident response team for further investigation and to determine if additional systems are affected.
+- Implement enhanced monitoring and logging for ScreenConnect and other remote access tools to detect similar activities in the future, ensuring that alerts are promptly reviewed and acted upon."""
+references = [
+    "https://www.netsupportsoftware.com/",
+]
+risk_score = 73
+rule_id = "5f73aef2-7abc-4fd9-ac0d-ab8ec3e13891"
+severity = "high"
+tags = [
+    "Domain: Endpoint",
+    "OS: Windows",
+    "Use Case: Threat Detection",
+    "Tactic: Command and Control",
+    "Resources: Investigation Guide",
+    "Data Source: Elastic Endgame",
+    "Data Source: Elastic Defend",
+    "Data Source: Sysmon",
+    "Data Source: SentinelOne",
+    "Data Source: Microsoft Defender for Endpoint",
+    "Data Source: Windows Security Event Logs",
+    "Data Source: Crowdstrike",
+]
+timestamp_override = "event.ingested"
+type = "eql"
+
+query = '''
+process where host.os.type == "windows" and event.type == "start" and
+ (process.name : "client32.exe" or ?process.pe.original_file_name == "client32.exe" or process.parent.name : "client32.exe") and
+ (
+  process.executable :
+               ("?:\\Users\\*.exe",
+                "?:\\ProgramData\\*.exe",
+                "\\Device\\HarddiskVolume?\\Users\\*.exe",
+                "\\Device\\HarddiskVolume?\\ProgramData\\*.exe") or
+  ?process.parent.executable : ("?:\\Users\\*\\client32.exe", "?:\\ProgramData\\*\\client32.exe")
```

Comment:
Do we need this condition? It seems like one we could remove

Reply:
yes we need it to detect processes spawned from client32.exe (NetSupport running from non std path)